horizon/templates/app-settings.configmap.yaml

```diff
@@ -11,8 +11,15 @@ data:
   DOMAIN: {{ include "core.domain" . | quote }}
   POSTGRES_HOST: {{ .Values.dependencies.postgresql.hostname | quote }}
   POSTGRES_PORT: {{ .Values.dependencies.postgresql.port | quote }}
+  POSTGRES_USER: {{ .Values.dependencies.postgresql.postgresUsername }}
+  POSTGRES_PASSWORD: {{ .Values.dependencies.postgresql.postgresPassword }}
   POSTGRES_SSL_MODE: {{ .Values.dependencies.postgresql.sslmode | quote }}
   POSTGRES_SSL_FACTORY: {{ .Values.dependencies.postgresql.sslfactory | quote }}
+  OPENNMS_DATABASE_CONNECTION_IDLETIMEOUT: {{ .Values.dependencies.postgresql.idleTimeout }}
+  OPENNMS_DATABASE_CONNECTION_LOGINTIMEOUT: {{ .Values.dependencies.postgresql.loginTimeout }}
+  OPENNMS_DATABASE_CONNECTION_MINPOOL: {{ .Values.dependencies.postgresql.minPool }}
+  OPENNMS_DATABASE_CONNECTION_MAXPOOL: {{ .Values.dependencies.postgresql.maxPool }}
+  OPENNMS_DATABASE_CONNECTION_MAXSIZE: {{ .Values.dependencies.postgresql.maxSize }}
```

Comment:
Can you investigate if we need `| quote`? When the template generates the configuration, it renders the value in `"25"` instead of `25`.